package.json

```diff
@@ -31,7 +31,7 @@
     "test:lockfile": "lockfile-lint -p package-lock.json -t npm -a npm -o https: -c -i",
     "test:lint": "eslint --ext js,jsx,md,snap .",
     "start": "node lib/server/index.js",
-    "start:inspect": "node --inspect --expose-gc lib/server/index.js",
+    "start:inspect": "node --inspect-brk --expose-gc lib/server/index.js",
```

Comment:
```suggestion
    "start:inspect": "node --inspect --expose-gc lib/server/index.js",
```

Reply:
resolved in 5072f48bf0080408e9584a2277514d411e3298ef